rts/System/EventHandler.cpp

```diff
@@ -59,6 +61,7 @@ void CEventHandler::SetupEvents()
 
 void CEventHandler::AddClient(CEventClient* ec)
 {
+	ZoneScoped;
```

Comment:
This is unlikely an expensive operation, do we really need it here?

Reply:
Yes, I was just happily adding them to ~all events, but here it doesn't make sense, it's just adding listener, dropped here and in removal.